FF1Lib/asm/1B_B000_StartWithRoutines.asm

```diff
@@ -94,8 +96,18 @@ NotANone2:
   PHP
   LDY tmp
   PLP
-  RTS 
-   
+  RTS
+
+IsEquipped:
+	@Loop:
+		LDA (btl_ob_charstat_ptr), Y
+		BPL @NoEquip
+			RTS
+	@NoEquip:
+		INY
+		CPY #$20
```

Comment:
might cause false positives with weapons

Reply:
Good point, I can add a better check.

Reply:
well it's more that if you're doing weapons you're also cycling armors, and then because the game use relative ids for weapons/armors, it might trigger on equipped armors if it's the same id as a weapon